### PR TITLE
add new character

### PR DESCRIPTION
**Pull request by xwang7**

### CODE REVIEW

**Comment by xwang7** [comment]
Please review this, @casey-collab

File: Chinese/xinwang/xin.md [hunk 1]
```diff
@@ -0,0 +1 @@
+小熊维尼又把头埋在罐子里闻了闻，说：“我记得有一个叔叔曾经告诉过我，说他看到奶酪过奶酪也是这种颜色的，那么也就是说这有可能是奶酪了喽？ 保险起见，还是让我先鉴定一下吧，恩，确实应该先鉴定一下。“
```
Comment (casey-collab) [inline Chinese/xinwang/xin.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: Chinese/xinwang/xin.md (hunk 1 above)
Comment (xwang7) [inline Chinese/xinwang/xin.md]: ok

**Comment by xwang7** [comment]
Please review this, @casey-collab

File: Chinese/xinwang/xin.md (hunk 1 above)
Comment (casey-collab) [inline Chinese/xinwang/xin.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by xwang7** [comment]
I made modification can you check@casey-collab

File: Chinese/xinwang/xin.md (hunk 1 above)
Comment (casey-collab) [inline Chinese/xinwang/xin.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by xwang7** [comment]
I made another update. can you check@casey-collab

**Comment by casey-collab** [comment]
Looks great! :night_with_stars:

**Pull request merged**